modules/ROOT/pages/_partials/commons/common-document.adoc

```diff
@@ -440,7 +440,19 @@ Database Changes::
 --
 [source, Kotlin, subs="attributes+"]
 ----
-include::android:example$snippets/app/src/main/kotlin/com/couchbase/code_snippets/FlowExamples.kt[tag="flow-as-database-change-listener"]
+fun watchQuery(query: Query): LiveData<List<Result>> {
```

Comment:
AKKK! Please... lets not put code in the adoc files.  I can change the source....